.gitignore

```diff
@@ -6,3 +6,4 @@ __pycache__/
 /*.egg-info/
 /build/
 /dist/
+.idea/
```

Comment:
The best place for personal settings is `~/.gitignore` as it applies to all repositories and doesn't pollute each repository with unrelated stuff.